### PR TITLE
chapter8

### PR DESCRIPTION
**Pull request by nnmai-3108**
# 1. Các route tương ứng trong Auth::route() là gì?

* Login Routes
>Route::get('login', [LoginController::class, 'showLoginForm'])->name('login');
Route::post('login', [LoginController::class,'login']);
Route::post('logout',  [LoginController::class,'logout'])->name('logout');>

* Registration Routes
>Route::get('register', [RegisterController::class, 'showRegistrationForm'])->name('register');
Route::post('register', [RegisterController::class, 'register']);>

* Password Reset Routes
>Route::get('password/reset', [ForgotPasswordController::class, 'showLinkRequestForm'])->name('password.request');
Route::post('password/email', [ForgotPasswordController::class, 'sendResetLinkEmail'])->name('password.email');
Route::get('password/reset/{token}', [ForgotPasswordController::class, 'showResetForm'])->name('password.reset');
Route::post('password/reset', [ForgotPasswordController::class, 'reset'])->name('password.update');>

* Confirm Password 
>Route::get('password/confirm', [ConfirmPasswordController::class, 'showConfirmForm'])->name('password.confirm');
Route::post('password/confirm', [ConfirmPasswordController::class, 'confirm']);>

* Email Verification Routes
>Route::get('email/verify', [VerificationController::class, 'show'])->name('verification.notice');
Route::get('email/verify/{id}/{hash}', [VerificationController::class, 'verify'])->name('verification.verify');
Route::get('email/resend',  [VerificationController::class, 'resend'])->name('verification.resend');>

# 2. Khi cần bổ sung logic login, phải sửa ở đâu?

Sửa trong Controllers\Auth\LoginController

### CODE REVIEW

**Comment by nnmai-3108** [comment]
ready

File: public/build/manifest.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "file": "assets/app.aac2dc08.js",
+    "src": "resources/js/app.js",
+    "isEntry": true
+  },
+  "resources/sass/app.scss": {
+    "file": "assets/app.c090eeea.css",
+    "src": "resources/sass/app.scss",
+    "isEntry": true
+  }
+}
```
Comment (ghost) [inline public/build/manifest.json]: eof

**Comment by nnmai-3108** [comment]
ready

**Review by ghost — APPROVED** [review]

File: public/build/assets/app.aac2dc08.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+var Ur=typeof globalThis<"u"?globalThis:typeof window<"u"?window:typeof global<"u"?global:typeof self<"u"?self:{};function SA(s){return s&&s.__esModule&&Object.prototype.hasOwnProperty.call(s,"default")?s.default:s}var _a={exports:{}};/**
```
Comment (nampt-3149) [inline public/build/assets/app.aac2dc08.js:1]: Cho mấy file mix này vào gitignore đi bạn.

**Comment by nnmai-3108** [comment]
ready

**Comment by huyenttm-0901** [comment]
merged

**Pull request merged**